### PR TITLE
feat(OpenGLPolyDataMapper): Allow to create callback on mapper

### PR DESCRIPTION
**Pull request by laurennlam**
User can use callback to update shaders' attributes.
@jpouderoux has participated to this pull request.

### CODE REVIEW

**Comment by jourdain** [comment]
@martinken any comment?

I would go with `ShaderCallbacks` instead of `ShaderCallBacks` assuming that make sense to have callback here.

**Comment by martinken** [comment]
If you could change it to be lowercase backs  e.g. setCallbacks that would better and match VTK.  Given that minor change +2, looks good

File: Sources/Rendering/OpenGL/PolyDataMapper/api.md [hunk 1]
```diff
@@ -127,5 +127,21 @@ mapperSpecificProp['OpenGL']['FragmentShaderCode'] = 'MyNewFragmentShaderCode';
...
 
+### CallBack:
+You may want to update some shader code by using callback :
 
+```js
+const mapperSpecificProp = mapper.getViewSpecificProperties();
+mapperSpecificProp.ShaderCallbacks = [];
+mapperSpecificProp.ShaderCallbacks.push({
+	userData: ...,
+	callBack: function(userData, cellBO, ren, actor) {
```
Comment (jourdain) [inline Sources/Rendering/OpenGL/PolyDataMapper/api.md]: Should that be `callback`.

File: Sources/Rendering/OpenGL/PolyDataMapper/index.js [hunk 2]
```diff
@@ -903,6 +903,13 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
     publicAPI.setPropertyShaderParameters(cellBO, ren, actor);
     publicAPI.setCameraShaderParameters(cellBO, ren, actor);
     publicAPI.setLightingShaderParameters(cellBO, ren, actor);
+
+    const listCallbacks = model.renderable.getViewSpecificProperties().ShadersCallbacks;
+    if (listCallbacks) {
+      listCallbacks.forEach((object) => {
+        object.callBack(object.userData, cellBO, ren, actor);
```
Comment (jourdain) [inline Sources/Rendering/OpenGL/PolyDataMapper/index.js]: Same here: `callback`

**Comment by laurennlam** [comment]
@jourdain I think it's ready to merge

**Pull request merged**